modules/Makefile.kubernetes

```diff
@@ -175,8 +175,8 @@ kubernetes\:run-job:
 
 ## Show job logs and return job exit code, requires SSH if ran outside Codefresh
 kubernetes\:job-logs:
-	@echo -e "INFO: Monitoring job $(KUBERNETES_APP) on cluster $(call yellow,$(CLUSTER))..."
-	$(KUBEUTIL) tail-job $(KUBERNETES_APP); \
+	@echo -e "INFO: Monitoring job $(KUBERNETES_APP)$(KUBERNETES_JOB_NAME_SUFFIX) on cluster $(call yellow,$(CLUSTER))..."
+	$(KUBEUTIL) tail-job $(KUBERNETES_APP)$(KUBERNETES_JOB_NAME_SUFFIX); \
```

Comment:
not an expert on bash concatenation of strings, if KUBERNETES_JOB_NAME_SUFFIX is empty

Reply:
but probably OK?